fluentd/configs.d/openshift/input-pre-k8s-audit-log.conf

```diff
@@ -0,0 +1,16 @@
+<source>
+  @type tail
+  @id k8s-audit-input
+  @label @INGRESS
+  @log_level debug 
+  path "#{ENV['K8S_AUDIT_FILE'] || '/var/log/kube-apiserver/audit.log'}"
+  pos_file "#{ENV['K8S_AUDIT_POS_FILE'] || '/var/log/kube-apiserver/audit.log.pos'}"
+  tag audit.log
+  <parse>
+    @type json
+    time_key requestReceivedTimestamp
```

Comment:
this means the record will contain a field called `requestReceivedTimestamp`?  I think you will need a record transformer to rename this to `time` or `@timestamp` - looks like the format is already rfc3339

The problem with record_transformer plugins is that they are very, very slow and will cause performance problems - it will probably be necessary to extend the viaq data model filter plugin to make `src_time_name` is a list, then add `requestReceivedTimestamp` to that list

Reply:
This has been moved to the cluster-logging-operator: https://github.com/jcantrill/cluster-logging-operator/pull/1

Reply:
would definitely still use some feedback there :)